scripts/regenerate.sh

```diff
@@ -19,13 +19,12 @@ WORKDIR=$(mktemp -d)
 
 function finish {
   rm -rf "$WORKDIR"
-  # Revert back the PATH to client's original value
-  export PATH=$ORIGINAL_PATH
 }
 trap finish EXIT
 
-GOBIN="${WORKDIR}"/bin
-ORIGINAL_PATH=$PATH
+source ./scripts/install-protoc.sh "${WORKDIR}"
```

Comment:
Why `source`?  Generally you don't want to `source` anything unless it's explicitly supposed to add stuff to your environment.

Reply:
Done.